packages/shared/lib/core/network/tests/network.test.ts

```diff
@@ -40,125 +44,91 @@ describe('File: network.ts', () => {
                 bech32Hrp: 'smr',
-                baseToken: BASE_TOKEN[NetworkProtocol.Shimmer],
+                minPowScore: 1500,
+                belowMaxDepth: 15,
                 rentStructure: {
-                    vByteCost: 500,
-                    vByteFactorData: 10,
-                    vByteFactorKey: 1,
+                    vByteCost: 100,
+                    vByteFactorData: 1,
+                    vByteFactorKey: 10,
                 },
+                tokenSupply: TokenSupply.Shimmer,
             },
-            [NetworkType.Devnet]: {
-                id: 'testnet',
-                name: 'Testnet',
-                protocol: NetworkProtocol.Shimmer,
-                type: NetworkType.Devnet,
+            baseToken: EXPECTED_BASE_TOKEN[NetworkProtocol.Shimmer],
+        },
+        [NetworkId.Testnet]: {
+            id: NetworkId.Testnet,
+            name: 'Testnet',
+            protocol: {
+                version: 2,
+                networkName: 'testnet',
                 bech32Hrp: 'rms',
-                baseToken: BASE_TOKEN[NetworkProtocol.Shimmer],
+                minPowScore: 1500,
+                belowMaxDepth: 15,
                 rentStructure: {
-                    vByteCost: 500,
-                    vByteFactorData: 10,
-                    vByteFactorKey: 1,
+                    vByteCost: 100,
+                    vByteFactorData: 1,
+                    vByteFactorKey: 10,
                 },
+                tokenSupply: TokenSupply.Testnet,
             },
-            [NetworkType.PrivateNet]: <INetwork>{
-                name: 'Private Net',
-                protocol: NetworkProtocol.Shimmer,
-                type: NetworkType.PrivateNet,
-            },
+            baseToken: EXPECTED_BASE_TOKEN[NetworkProtocol.Shimmer],
         },
     }
 
-    const OFFICIAL_NODE_URLS: Readonly<{ [key in NetworkProtocol]?: { [key in NetworkType]?: string[] } }> = {
-        [NetworkProtocol.IOTA]: {
-            [NetworkType.Mainnet]: [
-                'https://chrysalis-nodes.iota.org',
-                'https://chrysalis-nodes.iota.cafe',
-                'https://mainnet-node.tanglebay.com',
-            ],
-            [NetworkType.Devnet]: [
-                'https://api.lb-0.h.chrysalis-devnet.iota.cafe',
-                'https://api.lb-1.h.chrysalis-devnet.iota.cafe',
-            ],
-        },
-        [NetworkProtocol.Shimmer]: {
-            [NetworkType.Mainnet]: ['https://api.shimmer.network'],
-            [NetworkType.Devnet]: ['https://api.testnet.shimmer.network'],
-        },
+    const EXPECTED_NODE_URLS: Readonly<{ [key in NetworkId]?: string[] }> = {
+        [NetworkId.Iota]: [
+            'https://chrysalis-nodes.iota.org',
+            'https://chrysalis-nodes.iota.cafe',
+            'https://mainnet-node.tanglebay.com',
+        ],
+        [NetworkId.Shimmer]: ['https://api.shimmer.network'],
+        [NetworkId.Testnet]: ['https://api.testnet.shimmer.network'],
```

Comment:
Can we use the `OFFICIAL_NODE_URLS` constant?

Reply:
I guess you wanted to explicitly say these are the "expected" ones?

Reply:
Honestly this was just a quick rewrite of the existing tests, I am not sure why we add the values here. If anything we should just hard code the values for each test as this logic to get everything from enums is just a duplicate of the logic we are testing....